public/resources/ts/calculate-player-stats.ts

```diff
@@ -127,6 +127,16 @@ export function getPlayerStats() {
     }
   }
 
+  {
+    if (calculated.cakes) {
+      for (const cake of calculated.cakes) {
+        if (stats[cake.stat]) {
+          stats[cake.stat].cakes ??= 0;
```

Comment:
I would call them "century_cakes" here too so players can't confuse them with the cake bag